pkg/cnidel/cniconfs.go

```diff
@@ -49,13 +47,13 @@ func getSriovCniConfig(netInfo *danmtypes.DanmNet, ipamOptions danmtypes.IpamCon
   if err != nil {
     return nil, errors.New("failed to get the name of the sriov PF for device "+ ep.Spec.Iface.DeviceID +" due to:" + err.Error())
   }
-  vlanid := netInfo.Spec.Options.Vlan
   sriovConfig := SriovNet {
+    CNIVersion: "0.3.1",
```

Comment:
why are you hard-coding CNI version into the middle of the code, when it was already data structure driven?
the value should be just passed through the interface, now that we want to use it

Reply:
done